tests/_fish_completion.fish

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env fish
```

Comment:
Hmmmm... I'm thinking about this file. For me, it should be inside `external` or `samples`... but I don't have strong opinion about that. What do you think @MatheusBernardino ?

Reply:
I'll remove this file, and I'll summarize it into a string inside `fish_completion_test.sh`. It will also give more flexibility to this piece of code.